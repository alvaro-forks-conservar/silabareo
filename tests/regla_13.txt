aarónico
capea
recochinéate
boa
bajaes
arráez
lee
regodéese
áloe
bóer
tablao
feo
rubéola
moho
manía
ríe
lío
púa
acentúe
búho