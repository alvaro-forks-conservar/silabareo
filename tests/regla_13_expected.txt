a-a-ró-ni-co
ca-pe-a
re-co-chi-né-a-te
bo-a
ba-ja-es
a-rrá-ez
le-e
re-go-dé-e-se
á-lo-e
bó-er
ta-bla-o
fe-o
ru-bé-o-la
mo-ho
ma-ní-a
rí-e
lí-o
pú-a
a-cen-tú-e
bú-ho